RELEASENOTES.md

```diff
@@ -1 +1 @@
-
```

Comment:
as it is right now, I don't believe it does b/c I set the `--wait` to `true` so it will generate a sequential script. that being said we do need to set the default value of `--wait` back to `false` in the `generate-script` parallel PR (which i have already make a note for it 😃 )